caom2utils/caom2utils/tests/test_fits2caom2.py

```diff
@@ -106,6 +107,7 @@
 test_plugin_module = os.path.join(TESTDATA_DIR, 'test_plugin.py')
 test_class_plugin_module = os.path.join(TESTDATA_DIR, 'test_plugin_class.py')
 non_conformant_plugin_module = os.path.join(TESTDATA_DIR, 'nonconformant.py')
+testproxy = os.path.join(TESTDATA_DIR, 'test_proxy.pem')
```

Comment:
Where is this coming from?

Reply:
It's an expired cert that's used for test_get_vos_meta, meant to pass the location test in the net.Subject ctor.